woocommerce/woocommerce-gutenberg-products-block
# Mini Cart block: add footer (subtotal, cart & checkout buttons and payment method icons)

Fixes #4667 

This PR adds the footer including the subtotal, cart/checkout buttons, and Payment Method icons to the Mini Cart Drawer. This also updates the `Drawer` style to fix the overlap issue caused by the Storefront handheld footer bar.

Note:
- The design uses different text for Checkout links (styled as buttons) between mobile and desktop. I use the same button for all devices/screen sizes.
- The design doesn't provide a hover style for the Cart button. The default hover style looks good to me so I reuse it. I'm happy to change it if needed.




#### Accessibility




### Screenshots

<table>
<tr>
	<td>With scrollbar
	<td>Without scrollbar
<tr>
	<td><img src="https://user-images.githubusercontent.com/5423135/138453641-2caa2f87-4b10-423a-ad94-329274b4a4a2.png" />
	<td><img src="https://user-images.githubusercontent.com/5423135/138453739-504fffee-6167-4ed3-b7fe-b880d8f62a09.png" />
</table> 

Mobile:
![image](https://user-images.githubusercontent.com/5423135/138456301-49768001-29f7-4c69-8825-cf8ebe50b894.png)




### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Deactivate all active payment methods.
2. Check out this PR and build the assets.
3. Add Mini Cart block to a page and then add a product to cart.
4. See the Mini Cart Drawer open, see the new footer without any payment icons.
5. Try resizing the browser to see it matches the screenshot above.
6. Set up/enable [WooCommerce Stripe Payment Gateway](https://wordpress.org/plugins/woocommerce-gateway-stripe/).
7. Open the Mini Cart again, see the Payment provider icons like the design above.
